### PR TITLE
Revise Basics Chapter

### PR DESCRIPTION
**Pull request by Doishy**
Added more clarification for nomenculture.
Set IU as primary examples.
Changes to presentation of certain piles to reflect current practice.

### CODE REVIEW

File: markdown/chapters/meandeck/basics.md [hunk 1]
```diff
@@ -4,59 +4,110 @@ order: 0
...
+
+The first consideration to make is how you will *dig* into the pile.  
+We will use the phrase *dig* to mean to initially draw into the five cards you have stacked
+with Doomsday.  
+The most common way to *dig* into your pile is via a *cantrip* effect. This is a spell or
+ability which draws a card upon resolution. Typically this is provided by one of the commonly
+played {U} filter effects like {{Preordain}} or a free cycle effect like {{Street Wraith}}.
+If you are not under pressure you can also simply pass the turn to the opposing player and use
+your next turn's draw step in order to *dig* into the *pile*. When this is performed it is
+simply know as a *pass the turn pile* or *PTT pile*.
```
Comment (Lutri4eva) [inline markdown/chapters/meandeck/basics.md:41]: i believe it should known instead of know

Comment (Doishy) [reply markdown/chapters/meandeck/basics.md]: Fixed.

File: markdown/chapters/meandeck/basics.md [hunk 2]
```diff
@@ -4,74 +4,138 @@ order: 0
...
-Often enough, steps 2 and 3 go together by using a *cantrip* effect. This is
-typically a draw spell like Preordain or a free cycle effect like Street Wraith,
-which helps *dig* into the pile. This can be done in a single turn or spread
-over multiple turns, utilising your draw step as a way to *dig* into the pile.
+1. Draw into your pile enough so that the Oracle's trigger can win you the game
+1. Resolve Thassa's Oracle
+
+<row variant="centered">{{!DD}}{{!TO}}</row>
+
+Out these steps, we will start by detailing how to achieve steps 2. and 3. as
```
Comment (angrybacon) [inline markdown/chapters/meandeck/basics.md:31]: > Out these steps

Is that equivalent to "with these out of the way"? I wasn't sure whether that's an idiom or a typo 🤔

File: markdown/chapters/meandeck/basics.md [hunk 3]
```diff
@@ -4,74 +4,138 @@ order: 0
...
+In each example we shall present any mana requirements or card requirements
+followed by the pile as a visual display.
+
+If we take the following example:
 
-## Single Cantrip + Predict Pile
+> {B}{B}{B} + {U}{U} + Cantrip
 
-> {B}{B}{B} + {1}{U} + Cantrip
+This cost shows we require {B}{B}{B} for casting Doomsday. {U}{U} for casting
```
Comment (angrybacon) [inline markdown/chapters/meandeck/basics.md:69]: How about a level 2 heading around here that looks like "Pile Notation"?

Comment (Doishy) [reply markdown/chapters/meandeck/basics.md:71]: Added

File: markdown/chapters/meandeck/basics.md (hunk 2 above)
Comment (Doishy) [reply markdown/chapters/meandeck/basics.md]: Reworded, was tired, poorly written me's fault

**Review by angrybacon — APPROVED** [review]

**Pull request merged**